lib/pg.utils.js

```diff
@@ -103,6 +103,17 @@ const validateIdentifier = (name, type, prefix = '') => {
   }
 };
 
+const generateQueryParams = (count, start = 1) => {
+  let params = '';
+  for (let i = start; i < start + count; i++) {
+    if (i !== start) {
+      params += ', ';
+    }
+    params += `$${i}`;
+  }
```

Comment:
```params += (i === start ? '' : ', ') + `$${i}`;```

Reply:
I don't like using ternary when one of the branches of missing as it usually makes the statement more confusing ever though reducing line count.
Also, this function is a copy paste from @belochub pg provider PR.

Reply:
Ok